src/Provider/OpenRouteService/README.md

```diff
@@ -0,0 +1,27 @@
+# OpenRouteService Geocoder provider
+[![Build Status](https://travis-ci.org/geocoder-php/open-route-service-provider.svg?branch=master)](http://travis-ci.org/geocoder-php/open-route-service-provider)
+[![Latest Stable Version](https://poser.pugx.org/geocoder-php/open-route-service-provider/v/stable)](https://packagist.org/packages/geocoder-php/open-route-service-provider)
+[![Total Downloads](https://poser.pugx.org/geocoder-php/open-route-service-provider/downloads)](https://packagist.org/packages/geocoder-php/open-route-service-provider)
+[![Monthly Downloads](https://poser.pugx.org/geocoder-php/open-route-service-provider/d/monthly.png)](https://packagist.org/packages/geocoder-php/open-route-service-provider)
+[![Code Coverage](https://img.shields.io/scrutinizer/coverage/g/geocoder-php/open-route-service-provider.svg?style=flat-square)](https://scrutinizer-ci.com/g/geocoder-php/open-route-service-provider)
+[![Quality Score](https://img.shields.io/scrutinizer/g/geocoder-php/open-route-service-provider.svg?style=flat-square)](https://scrutinizer-ci.com/g/geocoder-php/open-route-service-provider)
+[![Software License](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square)](LICENSE)
+
+This is the OpenRouteService provider from the PHP Geocoder. This is a **READ ONLY** repository. See the
+[main repo](https://github.com/geocoder-php/Geocoder) for information and documentation. 
+
+### Install
+
+```bash
+composer require geocoder-php/open-route-service-provider
```

Comment:
I would put openrouteservice (without dashes) since it is how they refer to it (see https://github.com/GIScience/openrouteservice)